graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx

```diff
@@ -5,7 +5,17 @@ import lodash from 'lodash';
 import { Button, ButtonToolbar, Col, Row } from 'components/graylog';
 import { Input } from 'components/bootstrap';
 
-const FORM_FIELDS = ['firstName', 'lastName', 'email', 'company'];
+const FORM_FIELDS = ['firstName', 'lastName', 'email', 'phone', 'company'];
+
+const PHONE_PATTERN = /^[0-9+]+[0-9\-.\s]+$/;
```

Comment:
Should we use HTML5 [`pattern`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefpattern) to validate?